content/api/customer-alerts.apib

```diff
@@ -0,0 +1,314 @@
+FORMAT: 1A
+title: Customer-Alerts
```

Comment:
We shouldn't publicly call this `Customer-Alerts`. Replace with `Alerts`

Reply:
Done.